source/website/src/components/Translation.vue

```diff
@@ -479,13 +479,7 @@ export default {
     validTerminologyName: function() {
       const letterNumber = /^([A-Za-z0-9-]_?)+$/;
       // The name can be up to 256 characters long. Valid characters are a-z, A-Z, 0-9, -, and _.
-      if (this.customTerminologyCreateNew === "" || (this.customTerminologyCreateNew.match(letterNumber) && this.customTerminologyCreateNew.length<256)) {
-        return true;
-      }
-      else
-      {
-        return false;
-      }
+      return !!(this.customTerminologyCreateNew === "" || (this.customTerminologyCreateNew.match(letterNumber) && this.customTerminologyCreateNew.length<256));
```

Comment:
why `!!`?

Reply:
`!!` is a standard way in JavaScript to ensure a "truthy"/"falsy" value is converted to `true`/`false`, respectively.

SonarQube didn't like the original approach:

i.e.,
```
if (expression) {
  return true;
} else {
  return false;
}
```

I tried converting it to just...
```
return expression;
```

But, that introduced a bug because the expression can return `null` or something else that is "truthy"/"falsy" and some dependent code requires `true`/`false`. So, I had to do the `!!` thing to fix it.
```
return !!expression;
```